_rules/aria-required-accessible-name-gp8n89.md

```diff
@@ -0,0 +1,275 @@
+---
+id: gp8n89
+name: ARIA required accessible name
+rule_type: atomic
+description: |
+  This rule checks that WAI-ARIA accessible name is available when required.
+accessibility_requirements:
+  aria12:accessible_name_required:
+    title: ARIA 1.2, Accessible Name Required
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+  wcag20:1.1.1: # Non-text Content (A)
+    secondary: This success criterion is **related** to this rule. This is because elements assigned the ARIA role of `img` or `image` necessitate an accessible name according to WCAG standards. Some of the examples that either pass or fail overlap with this success criterion.
+  wcag20:1.3.1: # Info and Relationships (A)
+    secondary: This success criterion is **less strict** than this rule. Unlike WCAG, which might not mandate accessible names for certain elements like table elements, this rule requires them. Consequently, some of the failed examples satisfy this success criterion.
```

Comment:
Although I don't entirely agree with the interpretation, since there might be cases where something is visually clear due to the surrounding context or visual presentation of elements (e.g., a top promo banner carousel without an accessible name, but the section still requires one), I understand the rationale behind it being "related to."

Updating the secondary requirement.